pkg/iac-providers/arm/v1/load-file.go

```diff
@@ -50,6 +51,8 @@ func (a *ARM) LoadIacFile(absFilePath string) (allResourcesConfig output.AllReso
 		var config *output.ResourceConfig = &output.ResourceConfig{}
 
 		// TODO: make required calls to mapper API
+		newMapper := mapper.NewMapper("arm")
```

Comment:
Keeping the name short should be fine here, as the initializer is self-explanatory. 
```suggestion
		m := mapper.NewMapper("arm")
```